src/igraph/io/objects.py

```diff
@@ -577,34 +577,34 @@ def _export_graph_to_tuple_list(
 
     This function is the reverse of Graph.TupleList.
 
-    @param use_vids (bool): whether to label vertices in the output data
+    Example:
+
+      >>> g = Graph.Full(3)
+      >>> g.vs["name"] = ["apple", "pear", "peach"]
+      >>> g.es["name"] = ["first_edge", "second", "third"]
+
+      >>> # Get name of the edge
+      >>> g.to_tuple_list(edge_attrs=["name"])
+      [(0, 1, "first_edge"), (0, 2, "second"), (1, 2, "third")]
+
+      >>> # Use vertex names, no edge attributes
+      >>> g.to_tuple_list(use_vids=False)
+      [("apple", "pear"), ("apple", "peach"), ("pear", "peach")]
+
+    @param use_vids: (bool) whether to label vertices in the output data
       structure by their ids or their vertex_name_attr attribute. If
       use_vids=False but vertices lack a vertex_name_attr attribute, an
       AttributeError is raised.
-    @param edge_attrs (str or list of str): list of edge attributes to export
+    @param edge_attrs: (str or list of str) list of edge attributes to export
```

Comment:
Type annotations should be moved to the function header or removed. This one should be `Union[str, Sequence[str]]` as I guess it works for arbitrary sequences, not only lists.

Reply:
yep